CONTRIBUTING.md

```diff
@@ -33,7 +37,8 @@ Building and starting the backend API server requires the following tools:
 
 With these dependencies installed, you can proceed as follows:
 
-* Change to the `dev` folder and generate the local database by running the `populate.sh` script You will then be able to modify it on the fly.
+* Change to the `dev` folder and generate the local database by running the `populate.sh` script You will then be able
+  to modify it on the fly.
```

Comment:
Missing a period between "script" and "You"

Reply:
(Was already there but just noticed now)

Reply:
ah yes good catch, thanks !